function-server/pom.xml

```diff
@@ -0,0 +1,19 @@
+<project>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.github.dispatchframework.javabaseimage</groupId>
+    <artifactId>java-base-image</artifactId>
+    <version>1.0.0</version>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.undertow</groupId>
+            <artifactId>undertow-core</artifactId>
+            <version>2.0.3.Final</version>
```

Comment:
I would recommend making a variable for the versions. Typically right before the <dependencies> element you could add
```
<properties>
     <undertow.version>2.0.3.Final<undertow.version>
</properties>
```
Then you can use `${undertow.version}` in the version element.

Reply:
Sure I'll add that